Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
+        return self.field(field)
+    }
+}
+
+
+struct Artist {
+    let id: Int?
+    // Note the optionality. Fields that are marked `NOT NULL` in your schema should be non-optional.
+    var name: String
+    var genre: String?
+
+    // You probably want your own initializer
+    init(name: String, genre: String) {
+        self.id = nil
+        self.name = name
+        self.genre = genre
+    }
+}
+
+
+extension Artist: Table {
```

Comment:
I separated model and table. The table is just representable db layer. It defines schema. Fields, their db equivalents, foreign key, and in future - relations. 
It just give ability to write Table.field(.id) instead of string "id", which make code more strict. If we made a mistake - code wont compile. Model give one other layer. It links db and our structures. 
And ads convenient func like save, get and so on.